components/alert/alert.css

```diff
@@ -0,0 +1,81 @@
+@import "../../assets/common-css/sidebar.css";
+@import "../../assets/common-css/navbar.css";
+@import "../../assets/common-css/containers.css";
+@import "../../document-css/doc.css";
+
+.alert-head, .alert-topic{
+    color: var(--font-color);
+    font-size: 1.7rem;
+}
+.alert-head{
+    margin: -5px;
+}
+.alert-topic{
+    margin-top: 30px;
+}
+.alert-content{
+    padding: 20px;
+    font-size: 19px;
```

Comment:
You should try to use CSS variables with defined sizes (for rems)
allows for alot of reusability